content/en/docs/components/training/job-scheduling.md

```diff
@@ -1,7 +1,7 @@
 +++
 title = "Job Scheduling"
 description = "How to schedule a job with gang-scheduling"
-weight = 70
+weight = 60
                     
 +++
 {{% alert title="Out of date" color="warning" %}}
```

Comment:
Should we also remove Out of date banner ?

Reply:
Good point @andreyvelich

I would say that it's up to @tynama123 or @k82cn who proposed to maintain it instead of archiving. 

If you'd like to move it and review it later so you have more time is ok for me.

Reply:
Thanks.
If the content of the doc sounds for @tinyma123 and @k82cn, we can remove the Out of Date banner and move forward.

Reply:
Agree to move the OOD banner right now, it's already used by several users in product env :)